lava-v2-jobs-from-api.py

```diff
@@ -107,6 +107,8 @@ def get_job_params(config, test_config, defconfig, opts, build, plan_config):
     # hack for arm64 dtbs in subfolders
     if arch == 'arm64' and dtb:
         dtb = opts['dtb'] = os.path.basename(dtb)
+    if arch == 'riscv' and dtb:
+        dtb = opts['dtb'] = os.path.basename(dtb)
```

Comment:
That's not great but it was already a hack for `arm64`, it'll need to be improved in general with some code in the test configs.